### ISSUE TITLE
Docs: link to notebooks showing how to use pyvo

### ISSUE THREAD

**Issue by stargaser** [astropy/pyvo#210]
The documentation and/or the README could link to notebooks showing pyvo in action. The NAVO notebooks are [rendered as html](https://nasa-navo.github.io/navo-workshop) and available in [this repository ](https://github.com/NASA-NAVO/navo-workshop).

My recommendation is to link to these notebooks rather than to copy them into an examples subdirectory in the pyvo repository.

**Comment by funbaker** [comment]
Sure, it'd be a good thing to link several examples/tutorials into the documentation.

Not only one though, there's also the GAVO or ESA examples/tutorials (and maybe more as well).

**Comment by bsipocz** [comment]
Another reincarnation of the discussion is here: https://github.com/astropy/pyvo/issues/304

Shall we keep this idea on the table, or close it, or create a narrative docs page that links some of the notebook resources? (preferably the ones officially supported and maintained by the data providers, AND also CI tested, e.g. we can be sure they check the notebooks run at minimum with the latest pyvo and astropy).


@astropy/pyvo-maintainers what do you think?

**Comment by andamian** [comment]
Sorry, not sure how Is different from what it's being suggested in the in the ticket?

**Comment by bsipocz** [comment]
> Sorry, not sure how Is different from what it's being suggested in the in the ticket?

Since this issue was opened, we kind of said in relation of #304 of not to collect/track third party examples. So my suggestion was for a middle ground, collect, but only things actively supported and updated (e.g. things that have a working CI), so to avoid advertising stuff that may get out of date with newer versions of PyVO, etc.

**Comment by andamian** [comment]
I agree and this ticket suggest the same thing and I don't see a point in opening a new one. WRT we could ask 3rd parties that to show that they use CI to keep up with `PyVO` changes as a condition for having their examples linked.

**Comment by bsipocz** [comment]
Oh, I didn't mean to suggest opening a new one, but to close either this or #304 as the discussion and action plan kind of overlap.

**Comment by andamian** [comment]
Oh, I didn't realize that. In part because they are slightly different things. If we ask people to have their CIs use `PyVO` dev as a condition for the documentation link then that will address #304. But we could encourage others to use `PyVO` dev in their CIs even if they don't link documentation (`astroquery` is an example of that). That can be achieved by providing the CI recipe in the `README` for example.

**Comment by bsipocz** [comment]
Anyway, this (well, both aspects) is on my list for the next release, to add a docs page that points to notebooks and libraries that do test versions, etc.

**Comment by andamian** [comment]
Awesome! Thank you! I like the "Tabs" format in the `astroquery` README.

**Comment by bsipocz** [comment]
Tabs?

**Comment by bsipocz** [comment]
Ah, sure, we could/should do this here, too.
